### ISSUE TITLE
Model is not as described in the paper

### ISSUE THREAD

**Issue by durrantmm** [gussow/seeker#1]
In the bioRxiv paper, you describe the model as having 5 hidden LSTM layers. Looking at the models in this github repository, this is just not the case. The model is a single LSTM layer with 5 neurons. This is an absolutely tiny network. This has fewer than 200 parameters. Typical deep neural networks have tens of thousands to millions of parameters. You should take a close look at this.

**Comment by noamaus** [comment]
Our preprint reads "LSTMs with 5 hidden layers" which refer to  2 different models (hence LSTMs) and each has 5 hidden units. Perhaps 'units' is a more percise wording, but it is perfectly clear that we do not describe models with 5 LSTM cells, in which case the number 5 should appear before the word LSTMs. 
 Indeed our models are small, this is not a bug, but an advantageous feature. Smaller networks take shorter time to train and have lower risk of overfitting the training data. FYI, an important rule of thumb in machine learning (including deep learning of course) is the state of tradeoff between model fit and low complexity:
![Overfitting-underfitting-and-model-complexity_W640](https://user-images.githubusercontent.com/18428559/78918023-bd052680-7a5d-11ea-84ec-7d57964b4dec.jpg)
which is related to the Occam's razor principle. Since we obtained very good performance with our tiny models (that is maintained from training to varying test sets) there is quite little risk of underfitting, and a major reduction to the risk of overfitting (which is quite substantial given the dependencies in the data - a complex model can easily "memorize" and fail in new phage identification). 
It is true that many deep learning networks are very large, but it is a constraint not an advantage, for those more complex tasks with far more available independent training data. 
 
As this does not report a issue with the package but rather is about the paper, this is not the appropriate mechanism for this type of discussion, please use email in the future.
